airbyte-integrations/connectors/source-xero/acceptance-test-config.yml

```diff
@@ -42,11 +42,13 @@ acceptance_tests:
             bypass_reason: "Empty stream, further investigation is required"
         fail_on_extra_columns: false
   incremental:
+    # bypass_reason: "Xero only supports date-precision filtering and our sandbox does not have enough data to properly test incremental reads"
     tests:
       - config_path: "secrets/config.json"
         configured_catalog_path: "integration_tests/configured_catalog.json"
         future_state:
           future_state_path: "integration_tests/abnormal_state.json"
+        skip_comprehensive_incremental_tests: true # The incremental read is occuring in client side and not via API inject option
```

Comment:
Can you add more light about why we skip sequential read? It was added to fix your issue described here https://github.com/airbytehq/airbyte/pull/36956#issuecomment-2046706792? Probably it occurred due to second read with state from the first read returned same records as the first read, which we don't expect. Can you reproduce this behavior locally?

Reply:
Yes, I could reproduce this behavior locally,
With the help of client-side Incremental reads, TestTwoSequentialReads is passed
However, TestReadSequentialSlices is skipped due to this behavior.